g-api/config.yaml

```diff
@@ -102,6 +102,18 @@ behaviors:
         title: Text
         default: Hello
 contents:
+  player:
+    title: Player
+    inherited_interface: guillotina.interfaces.IItem
+    inherited_class: guillotina.content.Item
+    add_permission: guillotina.AddContent
+    properties:
+      team:
+        type: guillotina.schema.TextLine
+        title: Team
+      rank:
+        type: guillotina.schema.Int
+        title: Rank
   canvas:
     title: Canvas
     inherited_interface: guillotina.interfaces.IItem
```

Comment:
When I tried Grange on a personal project, I run it and tried to create a resource of type `My doc` and it didn't work. when you'll have a chance maybe you could double check if it's still working